cypress/integration/project.spec.js

```diff
@@ -165,14 +165,29 @@ describe('10 - Aplique elementos HTML de acordo com o sentido e propósito de ca
   });
 });
 
-describe('11 - Teste a semântica da sua página está aprovada pelo site achecker', () => {
-  it('Seu site deve passar sem problemas na verificação de semântica do site achecker', () => {
-    cy.readFile('./index.html').then((content) => {
-      cy.visit('https://achecker.ca/checker/index.php');
-      cy.contains('Paste HTML Markup').click();
-      cy.get('textarea').type(content);
-      cy.get('#validate_paste').click();
-      cy.contains('Congratulations! No known problems.');
-    });
+// describe('11 - Teste a semântica da sua página está aprovada pelo site achecker', () => {
+//   it('Seu site deve passar sem problemas na verificação de semântica do site achecker', () => {
+//     cy.readFile('./index.html').then((content) => {
+//       cy.visit('https://achecker.ca/checker/index.php');
+//       cy.contains('Paste HTML Markup').click();
+//       cy.get('textarea').type(content);
+//       cy.get('#validate_paste').click();
+//       cy.contains('Congratulations! No known problems.');
+//     });
+//   });
+// });
+
+describe.only('11 - Teste a semântica da sua página está aprovada pelo site achecker', () => {
+  before(() => {
+    // Continuum only needs to be set up once per testing context;
+    // the page under test can change without having to set up Continuum again
+    cy.setUpContinuum('continuum/continuum.conf.js')
+  })
+
+  it('Testing', () => {
+    cy.visit('http://127.0.0.1:5500')
+      .runAllAccessibilityTests()
```

Comment:
Dá forma como está é necessário estar rodando com o live server. Não seria melhor utilizar o código abaixo, como nos outros `it`s?
```suggestion
    cy.setUpContinuum('continuum/continuum.conf.js');
    cy.visit('/');
  });

  it('Testing', () => {
      cy.runAllAccessibilityTests()
```